src/Management/src/Endpoint/Metrics/Observer/CLRRuntimeObserver.cs

```diff
@@ -39,13 +40,14 @@ public class ClrRuntimeObserver : IRuntimeDiagnosticSource
     public ClrRuntimeObserver(IViewRegistry viewRegistry)
     {
         Meter meter = OpenTelemetryMetrics.Meter;
-        _ = meter.CreateObservableGauge("clr.memory.used", GetMemoryUsed, "Current CLR memory usage", "bytes");
-        _ = meter.CreateObservableGauge("clr.gc.collections", GetCollectionCount, "Garbage collection count", "count");
+        meter.CreateObservableGauge("clr.memory.used", GetMemoryUsed, "Current CLR memory usage", "bytes");
+        meter.CreateObservableGauge("clr.gc.collections", GetCollectionCount, "Garbage collection count", "count");
 
-        _ = meter.CreateObservableGauge("clr.threadpool.active", GetActiveThreadPoolWorkers, "Active thread count", "count");
-        _ = meter.CreateObservableGauge("clr.threadpool.avail", GetAvailableThreadPoolWorkers, "Available thread count", "count");
+        meter.CreateObservableGauge("clr.threadpool.active", GetActiveThreadPoolWorkers, "Active thread count", "count");
+        meter.CreateObservableGauge("clr.threadpool.avail", GetAvailableThreadPoolWorkers, "Available thread count", "count");
 
-        _ = meter.CreateObservableGauge("clr.process.uptime", GetUpTime, "Process uptime in seconds", "count");
+        meter.CreateObservableGauge("clr.process.uptime", GetUpTime, "Process uptime in seconds", "count");
+        meter.CreateObservableGauge("clr.cpu.count", () => Environment.ProcessorCount, "Total processor count", "count");
```

Comment:
In this case it would be to indicate to future readers that we recognize that `CreateObservableGauge` returns something and that we don't care about it, and we are intentionally throwing it away (as opposed to potentially being unaware of the existence of it)

Reply:
This  seems to be fairly common practice for  example see usage [here](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/c49a261d43a161b53ae41304fa1d8b2c0f2fa418/src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs#L41).

Reply:
I added the discards as part of #1049, because the return values were assigned to unused private fields, which I removed in that PR. I'm not against removing the discards if you believe that's better, just thought my changes were accidentally overwritten during the merge.